Sources/StreamChat/Database/DTOs/ChannelDTO.swift

```diff
@@ -126,13 +126,21 @@ extension NSManagedObjectContext {
         dto.isFrozen = payload.isFrozen
         dto.cooldownDuration = payload.cooldownDuration
 
+        if let teamId = payload.team {
+            let teamDTO = try saveTeam(teamId: teamId)
+            teamDTO.channels.insert(dto)
+            dto.team = teamDTO
```

Comment:
Is the `ChatChannel.team` value tested somewhere?

Reply:
oh my zshell you are right! I forgot to uncomment the property on the model. Thanks 🍻